src/index.js

```diff
@@ -2,12 +2,16 @@ import React from 'react';
 import ReactDOM from 'react-dom';
```

Comment:
![giphy (2)](https://user-images.githubusercontent.com/93735137/155985020-74eeb9e7-c953-4e4d-aab6-6b5caff7ef01.gif)
Hi @Nuri1977,
You did a great job!👏🏼 👏🏼
No linter errors. 
Added directory for redux files💪🏻
Redux store configured corectly.
Redux store configured correctly 
Action types are correct and the reducer works properly.
Items are rendered properly.
Good commit messages!
+1 Good job!